src/utils/general.tsx

```diff
@@ -35,3 +36,12 @@ export const parsePrice = (stringPrice: string) => {
 export const getSamplePage = (sku: string) => {
   return `${MAIN_SHOP_URL}sample/${sku}`;
 };
+
+export const getTrackingUrl = (trackingCode: string) => {
+  return `https://www.fedex.com/fedextrack/?action=track&trackingnumber=${trackingCode}&cntry_code=us&locale=en_US`;
+};
+
+export const isTrackingPackageButtonDisabled = (order: OrderX) => {
```

Comment:
For now, I think we should just check if the order.shipment.tracking field exists. I'm not sure how the order statuses will interfere with this, but just checking for the field should be a bit safer, as maybe the awb will be available before the status is changed to `shipped`.

Also, why is this function called `is{}Disabled` but it returns true when we want to show the button?